tests/functional/reporting-non-auth.js

```diff
@@ -287,6 +287,24 @@ define(
           .end();
       },
 
+      "clicking text area clears default value": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findByCssSelector("#steps_reproduce")
+          .click()
+          .end()
+          .sleep(500)
+          .findByCssSelector("#steps_reproduce")
```

Comment:
After `click()`, the context is still the textarea element, so you can just do something like this:

```js
.findByCssSelector("#steps_reproduce")
.click()
.sleep(500)
.getProperty("value")
...
```

No need to re-find the same DOM element.